solutions/instances/catalogValidationValues.json.template

```diff
@@ -1,6 +1,6 @@
 {
   "ibmcloud_api_key": $VALIDATION_APIKEY,
   "resource_group_name": $PREFIX,
-  "existing_kms_guid": $HPCS_US_SOUTH_GUID,
+  "existing_kms_guid": $HPCS_US_SOUTH_CRN,
```

Comment:
Firstly, `existing_kms_guid` need to change to `existing_kms_crn` now right? Secondly, `$HPCS_US_SOUTH_CRN` is not a support variable substitution value (see full list [here](https://github.ibm.com/GoldenEye/documentation/blob/master/catalog.md#4-set-up-the-validation)). @maheshwarishikha catalog pipeline might need to be updated to support this

Reply:
I don't see any `HPCS_US_SOUTH_CRN` variable in the above list.

Reply:
ok, I'll do this

Reply:
done, changes added